.github/workflows/mac-debug-build.yml

```diff
@@ -0,0 +1,187 @@
+name: ODBC Driver for Mac (Debug)
+
+on:
+  push:
+    paths:
+      - '.github/workflows/win-build.yml'
```

Comment:
`mac-debug-build.yml`?

Reply:
Addressed in https://github.com/Bit-Quill/arrow/pull/7#discussion_r1452813092